src/main/scala/com/datasonnet/Mapper.scala

```diff
@@ -90,32 +91,60 @@ object Mapper {
     (name -> jsonnetLibrary)
   }
 
+  def getParams(name: String, mimeType: String, header: Header): util.Map[String, Object] = {
+    if (header != null) {
+      val dfParams = header.getDataFormatParameters().get(mimeType);
+      if (dfParams != null) {
+        val mergedParams = dfParams.getOrDefault(Header.DATAFORMAT_DEFAULT, new util.HashMap[String, Object]())
+        mergedParams.putAll(dfParams.getOrDefault(name, new util.HashMap[String, Object]()))
+        mergedParams
+      } else new util.HashMap[String, Object]()
+    } else new util.HashMap[String, Object]()
+  }
+
   // TODO: will we require a header (or have a default header) on next release? I believe we should, as it is our last chance for a big required change, in which case
```

Comment:
You can go ahead and strip these comments, they're obviated by this PR

Reply:
Done